github/data_source_github_repository.go

```diff
@@ -37,6 +37,10 @@ func dataSourceGithubRepository() *schema.Resource {
 				Type:     schema.TypeBool,
 				Computed: true,
 			},
+			"visibility": {
+				Type:     schema.TypeBool,
```

Comment:
ah sorry I missed this! I believe we want `TypeString` here. the test files look fine, just would need to update this in the resource as well

Reply:
Welp.